app/main.py

```diff
@@ -1,2 +1,71 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int =8,
+            load_factor: float =0.666
+    ) -> None:
+        self.capacity = initial_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.table = [None] * self.capacity
+
+
+    class Node:
+        def __init__(self, key: Any, value: Any) -> None:
+            self.key = key
+            self.value = value
+            self.hash = hash(key)
+            self.next = None
```

Comment:
Consider defining the `Node` class outside of the `Dictionary` class. This will improve readability and maintainability, especially if the `Node` class is used in other parts of the code.